.circleci/config.yml

```diff
@@ -17,10 +17,18 @@ jobs:
   install_and_build:
     working_directory: ~/stockflux
     docker:
-      - image: circleci/node:10.15.1
+      - image: circleci/node:10.16.3
     steps:
       - checkout:
           path: ~/stockflux
+      - run:
+          name: Upgrade NPM to 6.11.3
+          command: sudo npm i -g npm@6.11.3
+      - run:
+          name: Node.js and NPM versions
+          command: |
+            node -v
+            npm -v
```

Comment:
Is this NPM upgrade step necessary? Also not sure if we need to print node and npm versions out.